spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/VolumetricConvolution.scala

```diff
@@ -203,24 +254,18 @@ class VolumetricConvolution[T: ClassTag](
         s" (${ nOutputPlane }x${ outputDepth }x${ outputHeight }x${ outputWidth })." +
         s" Output size is too small")
 
-    require(weight.dim() == 2 || weight.dim() == 5,
```

Comment:
why remove this check?

Reply:
I did not remove it. It is checked outside this function.